src/mobile/src/ui/theme/general.js

```diff
@@ -54,3 +54,34 @@ export function getBackgroundColor(screen, theme, inactive = false) {
     };
     return screenMap[screen];
 }
+
+export function getBackgroundColor(screen, theme) {
+    const { bar, body } = theme;
+    const screenMap = {
```

Comment:
Perhaps we should avoid this duplication by defining `screenMap` globally and then transforming it for wherever needed. It's also used in [getBorderColor](https://github.com/iotaledger/trinity-wallet/blob/develop/src/mobile/src/ui/theme/general.js#L27).